### ISSUE TITLE
Create a test that calls mpc contract from another contract

### ISSUE THREAD

**Issue by volovyks** [near/mpc-recovery#434]
### Description

We need to make sure that this flow works and that we do not confuse signer with predecessor, etc.

**Comment by volovyks** [comment]
@ChaoticTempest do you think it's still needed? Should we close it?

**Comment by ChaoticTempest** [comment]
Yeah, I think a simple test in integration side of things would be well worth. Just so we can test the user experience of this directly ourselves
